charts/shoot-core/components/charts/kube-proxy/templates/kube-proxy-daemonset.yaml

```diff
@@ -114,3 +132,7 @@ spec:
       - name: kernel-modules
         hostPath:
           path: /lib/modules
+      - name: kube-proxy-init-script
```

Comment:
Can you add a checksum annotation to the pod template for this configmap mount?

Reply:
Sure also thought about that, but i come to the conclusion that we won't change this script. What is the benefit?

Reply:
Can you look into the future 🔮 😃? OK, joking aside, you're right, it's probably more a theoretical case in which the annotation would help. I'm thinking about this script having a bug that will make the init container crash loop. Without the checksum annotation fixing the script won't help as it wouldn't trigger a roll-out. As said, probably very unlikely, but I'd still add the checksum, just to be safe (it doesn't harm having it, does it?).

Reply:
No, will add it. :)